components/admin_console/admin_definition.jsx

```diff
@@ -1287,6 +1287,15 @@ export default {
                                 needsUtils.stateValueFalse('LdapSettings.EnableSync'),
                             ),
                         },
+                        {
+                            type: Constants.SettingsTypes.TYPE_BOOL,
+                            key: 'LdapSettings.UseMemberOfOverlay',
+                            label: t('admin.ldap.memberOverlayTitle'),
+                            label_default: 'Use \'memberOf\' Overlay:',
+                            help_text: t('admin.ldap.memberOverlayDesc'),
+                            help_text_default: 'Set this to false only if the AD/LDAP instance does not support [reverse group membership maintenance](!https://www.openldap.org/doc/admin24/overlays.html#Reverse%20Group%20Membership%20Maintenance) using the \'memberOf\' overlay or if there is a known reason not to use it. Setting this to false will make group syncs slower.',
```

Comment:
Proposing adding some additional information to the help text - noting it as optional and explaining more about why it would need to be set.  We can modify the text if my interpretation of the linked resource was not quite right:

(Optional) This setting is used with the Group Filter to determine if group membership changes are updated on the user 'memberOf' attribute or by modification of the group object members attribute.  Set this to false only if the AD/LDAP instance does not support [reverse group membership maintenance](!https://www.openldap.org/doc/admin24/overlays.html#Reverse%20Group%20Membership%20Maintenance) using the \'memberOf\' overlay or if there is a known reason not to use it. Setting this to false will make group syncs slower.'

Reply:
I like that text more.  

I'm just wondering if it is actually optional the way it's currently presented?